docs/ides.md

```diff
@@ -0,0 +1,55 @@
+# IDEs
+
+The following desktop IDEs have been tested with Coder, though any IDE with SSH
+support should work:
+
+- VS Code (with [Remote -
+  SSH](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh)
+  extension)
+- JetBrains (with
+  [Gateway](https://www.jetbrains.com/help/idea/remote-development-a.html#launch_gateway)
+  installed)
+  - IntelliJ IDEA
+  - CLion
+  - GoLand
+  - PyCharm
+  - Rider
+  - RubyMine
+  - WebStorm
+
+## SSH configuration
+
+To access Coder via SSH, run the following in the terminal:
+
+```console
```

Comment:
TIL about `console`! I've always used `shell` but I guess they're [the same](https://github.com/highlightjs/highlight.js/blob/main/SUPPORTED_LANGUAGES.md)!